src/prof.c

```diff
@@ -131,6 +131,29 @@ static int		prof_dump_fd;
 /* Do not dump any profiles until bootstrapping is complete. */
 static bool		prof_booted = false;
 
+
+typedef struct prof_prod_cons_data_s prof_prod_cons_data_t;
+typedef struct prof_prod_cons_key_s prof_prod_cons_key_t;
+
+typedef struct prof_prod_cons_data_s {
+	uint64_t bytes;
+	uint64_t objs;
+	uint64_t cons_uid;
+	uint64_t prod_uid;
+} prof_prod_cons_data_t;
+
+typedef struct prof_prod_cons_key_s {
+	tsd_t *prod;
+	tsd_t *cons;
+} prof_prod_cons_key_t;
+
+/*
+ * maps producer-consumer pair (i.e. [prof_prod_cons_key_t]) to bytes and object
+ * counts (i.e. [prof_prod_cons_data_t])
+ */
+static ckh_t		prod_cons;
```

Comment:
No big deal either way, but we're mostly not aligning variable names (the clang-format diff is incoming, so this is neither here nor there).